src/AppRoutes.jsx

```diff
@@ -1,9 +1,10 @@
 import { React, useContext } from "react";
```

Comment:
You don’t need to explicitly import React in newer versions of React (v17+). You can remove React from the import statement unless it's needed for JSX.

Reply:
Thanks for the comment I will remove it!